src/barril/units/posc.py

```diff
@@ -13317,6 +13318,116 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
         f_unit_to_base,
         default_category=None,
     )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 1.0, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 1.0, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degC/Pa",
+        "degC/Pa",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 0.00001, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 0.00001, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degC/bar",
+        "degC/bar",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 1.0e6, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 1.0e6, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degC/MPa",
+        "degC/MPa",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 0.00001, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 0.00001, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "K/bar",
+        "K/bar",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 1.0, 0.0)
```

Comment:
should these be `MakeCustomaryToBase(0.0, 1.0, 1.0e-6, 0.0)`?

Reply:
Oh, my bad! Thanks for point that.

Reply:
I think we could start writing tests for these new units added in `barril`. Perhaps add a `test_unit_conversion.py` and add a test for this new category and do the (sanity) unit conversions check. What dou think @tadeu ?

Reply:
I agree!

Reply:
Here you can find some example of tests for units:
https://github.com/ESSS/barril/blob/2b79ba853bcfbb7599ae2458da71d6dc4d1cc00b/src/barril/units/_tests/test_posc.py#L99

Reply:
> Here you can find some example of tests for units:
> https://github.com/ESSS/barril/blob/2b79ba853bcfbb7599ae2458da71d6dc4d1cc00b/src/barril/units/_tests/test_posc.py#L99

Nice, thanks!

Reply:
@arthursoprana @tadeu @lincrosenbach tests added to `test_posc.py`.